src/main/java/com/vk/tlschema/highlighting/TLSchemaAnnotator.java

```diff
@@ -10,20 +12,20 @@
 import com.vk.tlschema.psi.*;
 import com.vk.tlschema.psi.impl.*;
 import com.vk.tlschema.search.TLSchemaSearchUtils;
-import org.bouncycastle.util.Arrays;
 import org.jetbrains.annotations.NotNull;
 
 import java.util.List;
 
 public class TLSchemaAnnotator implements Annotator {
     @Override
     public void annotate(@NotNull final PsiElement element, @NotNull AnnotationHolder holder) {
-        if (element instanceof TLSchemaResultTypeImpl) {
-            ColorType(((TLSchemaResultTypeImpl) element).getBoxedTypeIdent(), holder, false, true);
+        if (element instanceof TLSchemaResultTypeImpl schemaResultType) {
```

Comment:
Maybe switch case?

Reply:
`Pattern Matching for switch` became available only in [Java version 21](https://openjdk.org/jeps/441)

Reply:
Java...